pom.xml

```diff
@@ -14,8 +14,29 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.amazonaws</groupId>
+        <artifactId>aws-java-sdk-bom</artifactId>
+        <version>1.11.67</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
let's not do this.  it breaks my `make` style of building, and I don't ultimately think it wins anything over just doing straight maven deps.  remove this "import" dep and specficy the `1.11.67` version in sdk-core.   

Errrr.  Are you actually using `sdk-core`?  It doesn't seem to be actually listed as a dep for the kinesis jar.